easybuild/tools/filetools.py

```diff
@@ -2042,7 +2050,8 @@ def find_flexlm_license(custom_env_vars=None, lic_specs=None):
             if lic_files:
                 for lic_file in lic_files:
                     try:
-                        open(lic_file, 'r')
+                        # just try to open file for reading, no need to actually read it
+                        open(lic_file, 'r').close()
```

Comment:
Use `_open_wrapper`?

Reply:
Changed to 'rb'-mode and hence not using the function intended for text files

Reply:
:+1: